examples/forest_fire.jl

```diff
@@ -36,7 +36,8 @@ function forest_step!(forest)
     if length(forest.space.agent_positions[node]) == 0  # the cell is empty, maybe a tree grows here?
       p = rand()
       if p <= forest.properties[:p]
-        treeid = forest.agents[end].id +1
+        bigest_id = maximum([a.id for a in values(forest.agents)])

```

Comment:
this should be replaced with `maximum(keys(forest.agents))`, no? Since the key is always the agent id.